Not clear how to run the lint/fix locally, and from this result I cannot tell what change is needed:

```
##[error]  41:18  error  Replace `⏎······`request·${url}·with·target·audience·${targetAudience}`⏎····` with ``request·${url}·with·target·audience·${targetAudience}``  prettier/prettier

✖ 1 problem (1 error, 0 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.
```

I see the docs test failure, but I don't see other references within this repo and thus do not know where I need to make other changes:
```
[404] https://github.com/googleapis/google-auth-library-nodejs/blob/master/samples/idtokens-serverless.js


http://localhost:5758/
  [404] https://github.com/googleapis/google-auth-library-nodejs/blob/master/samples/idtokens-serverless.js
ERROR: Detected 1 broken links. Scanned 51 links in 1.649 seconds.
```

--- a/samples/idtokens-serverless.js
+++ b/samples/idtokens-serverless.js
@@ -38,9 +38,7 @@ function main(
       const {URL} = require('url');
       targetAudience = new URL(url).origin;
     }
-    console.info(
-      `request ${url} with target audience ${targetAudience}`
-    );
+    console.info(`request ${url} with target audience ${targetAudience}`);
     const client = await auth.getIdTokenClient(targetAudience);
     const res = await client.request({url});
     console.info(res.data);

